src/pages/home/HomeBlogPreview.jsx

```diff
@@ -103,7 +104,7 @@ function DesktopBlogPreview({ featuredPosts, allPosts }) {
               top: style.spacing.HEADER_HEIGHT + 20,
             }}
           >
-            <FeaturedBlogPreview blogs={featuredPosts} />
+            <FirstBlogPreview blogs={FirstPost} />
```

Comment:
```suggestion
            <FeaturedBlogPreview blogs={firstPost} />
```
...and rewrite this like this